scripts/globals/spells/bluemagic/hysteric_barrage.lua

```diff
@@ -24,24 +24,25 @@ end
 function onSpellCast(caster,target,spell)
     local params = {}
     -- This data should match information on http://wiki.ffxiclopedia.org/wiki/Calculating_Blue_Magic_Damage
-        params.tpmod = TPMOD_DAMAGE
-        params.damageType = tpz.damageType.HTH
-        params.scattr = SC_DETONATION
-        params.numhits = 5
-        params.multiplier = 1.25
-        params.tp150 = 1.625
-        params.tp300 = 1.75
-        params.azuretp = 1.875
-        params.duppercap = 80 -- D upper >=69
-        params.str_wsc = 0.0
-        params.dex_wsc = 0.3
-        params.vit_wsc = 0.0
-        params.agi_wsc = 0.0
-        params.int_wsc = 0.0
-        params.mnd_wsc = 0.0
-        params.chr_wsc = 0.0
+    params.tpmod = TPMOD_DAMAGE
+    params.attackType = tpz.attackType.PHYSICAL
+    params.damageType = tpz.damageType.HTH
```

Comment:
Because down here, we _are_ using the "hth" damage type, when for all the "hand-to-hand" (as identified on bgwiki) before this were simply "blunt".

Reply:
I grabbed the H2H classification from here: https://ffxiclopedia.fandom.com/wiki/Hysteric_Barrage

Reply:
Seems to be H2H here also: https://www.bg-wiki.com/bg/Hysteric_Barrage